.travis.yml

```diff
@@ -10,7 +10,7 @@ cache:
     - $HOME/.gradle/wrapper/
 
 # Enabling container based infrastructure hoping it will help the build speed <= this didn't work well, so it's reverted
-# see http://docs.travis-ci.com/user/workers/container-based-infrastructure/
+# see https://docs.travis-ci.com/user/migrating-from-legacy/
```

Comment:
@TimvdLippe Good idea. I will evaluate a stability and performance in https://github.com/mockito/mockito/pull/559